homeassistant/loader.py

```diff
@@ -1187,3 +1187,55 @@ def _lookup_path(hass: HomeAssistant) -> list[str]:
 def is_component_module_loaded(hass: HomeAssistant, module: str) -> bool:
     """Test if a component module is loaded."""
     return module in hass.data[DATA_COMPONENTS]
+
+
+@callback
+def async_get_issue_tracker(
+    hass: HomeAssistant | None,
+    *,
+    integration_domain: str | None = None,
+    module: str | None = None,
+) -> str:
+    """Return a URL for an integration's issue tracker."""
+    issue_tracker = (
+        "https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue"
+    )
+    if integration_domain is None and module is None:
+        # If we know nothing about the entity, suggest opening an issue on HA core
+        return issue_tracker
+
+    if hass and integration_domain is not None:
```

Comment:
Is "" a valid integration? 
Probably not so we can simplify the check

```suggestion
    if hass and integration_domain:
```